assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js

```diff
@@ -95,46 +135,47 @@ function DetailsGallery({
     : null;
 
   return (
-    <>
-      <SmallDisplayPagination>
-        {PrevButton}
-        {NextButton}
-      </SmallDisplayPagination>
-      <ColumnContainer>
-        <Column>
-          <LargeDisplayPagination>{PrevButton}</LargeDisplayPagination>
-          <CardGallery
-            story={template}
-            isRTL={isRTL}
-            galleryLabel={__('Template details by page', 'web-stories')}
-          />
-        </Column>
-        <Column>
-          <DetailContainer>
-            <Title>{template.title}</Title>
-            <ByLine>{byLine}</ByLine>
-            <Text>{template.description}</Text>
-            <MetadataContainer>
-              {template.tags.map((tag) => (
-                <Pill
-                  name={tag}
-                  key={tag}
-                  disabled
-                  onClick={() => {}}
-                  value={tag}
-                >
-                  {tag}
-                </Pill>
-              ))}
-            </MetadataContainer>
-            <MetadataContainer>
-              <ColorList colors={template.colors} size={30} />
-            </MetadataContainer>
-          </DetailContainer>
-          <LargeDisplayPagination>{NextButton}</LargeDisplayPagination>
-        </Column>
-      </ColumnContainer>
-    </>
+    <ColumnContainer>
+      <Column>
+        <PaginationContainer rightPadding>{PrevButton}</PaginationContainer>
+        <CardGallery
+          story={template}
+          isRTL={isRTL}
+          galleryLabel={__('Template details by page', 'web-stories')}
+        />
+      </Column>
+      <Column>
+        <DetailContainer>
+          <Display
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.MEDIUM}
+            as="h3"
+            data-testid="template-details-title"
+          >
+            {template.title}
+          </Display>
+          <ByLineText size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.MEDIUM}>
+            {byLine}
+          </ByLineText>
+          <DescriptionText
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.MEDIUM}
+          >
+            {template.description}
+          </DescriptionText>
+
+          <MetadataContainer>
+            {template.tags.map((tag) => (
+              <TemplateTag key={tag} disabled>
+                {tag}
+              </TemplateTag>
+            ))}
+          </MetadataContainer>
+          <MetadataContainer>
+            <ColorList colors={template.colors} size={30} />
```

Comment:
I think the colors need to be 32 px 
![image](https://user-images.githubusercontent.com/22185279/110032762-c2aecf00-7cf5-11eb-8aae-fba4f931d34c.png)

Reply:
thanks! good catch.